docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -478,6 +541,54 @@ The treatment algorithm is described in this :download:`word document found here
 
 **How to assign treatment effects:**
 
+Each treatment category has a hazard ratio associated with it both for progression-free survival and overall survival relative to the overall progression-free survival and overall survival of their demographic group as a whole. There are separate hazard ratios for the first line of treatment and later lines of treatment for each treatment category. Additionally, the hazard ratios for the isatuxamib- and daratumumab-containing treatment categories vary based on retreatment status (:code:`retreatment == True`). The hazard ratios are shown in the tables below.
+
+.. list-table:: First line of treatment hazard ratios
+   :header-rows: 1
+
+   * - Treatment category
+     - Progression free survival HR (95% UI)
+     - Overall survival HR (95% UI)
+   * - All categories combined
+     - 1 (reference)
+     - 1 (reference)
+   * - Isatuxamib-containing treatment category
+     - 0.429 (0.368, 0.495)
+     - 0.760 (0.645, 0.895)
+   * - Daratumumab-containing treatment category
+     - 0.429 (0.368, 0.495)
+     - 0.760 (0.645, 0.895)
+   * - Residual treatment category
+     - 1.00581 (1.0051, 1.0064)
+     - 1.0024 (1.0011, 1.0036)
+
+.. list-table:: Second and later lines of treatment hazard ratios
+   :header-rows: 1
+
+   * - Treatment category
+     - Progression free survival HR (95% UI)
+     - Overall survival HR (95% UI)
+   * - All categories combined
+     - 1 (reference)
+     - 1 (reference)
+   * - Isatuxamib-containing treatment category, :code:`retreated != True`
+     - 0.530 (0.356, 0.803)
+     - 1.159 (1.044, 1.185)
```

Comment:
I am surprised that this is >1

Reply:
This is above one in reference to Flatiron, but it is below one in reference to the residual category.